docs/cargo_features.md

```diff
@@ -31,6 +31,7 @@
 |flac|FLAC audio format support. It's included in bevy_audio feature.|
 |wav|WAV audio format support.|
 |vorbis|Vorbis audio format support.|
+|wasm_audio|WASM audio support.|
```

Comment:
As you said mp3 doesn't work, could you mention here which format does?

Reply:
Done. They should all work except mp3. Rodio is working on using Symphonia as an alternative backend, which might resolve this.